incompatibility/incompatibility-report/incomp-scanning_test.go

```diff
@@ -45,16 +47,49 @@ func TestBasicSecurityIncompatibility(t *testing.T) {
 	path1 := "../../generator/testfiles/other.yaml"
 	path2 := "../../examples/petstore/petstore.yaml"
 
-	var serversTest = []struct {
+	var securityTest = []struct {
 		path           string
 		expectSecurity bool
 	}{
 		{path1, false},
 		{path2, true},
 	}
-	for _, tt := range serversTest {
+	for _, tt := range securityTest {
 		if incompatibilityCheck(generateDoc(t, tt.path), IncompatibiltiyClassification_Security) != tt.expectSecurity {
 			t.Errorf("Incorrect security detection for file at %s, got %t\n", path1, tt.expectSecurity)
 		}
 	}
 }
+
+func TestIncompatibilityExistence(t *testing.T) {
+	petstorePath := "../../examples/petstore/petstore.yaml"
```

Comment:
declare inline to prevent accidental usage.

Reply:
fixed